api/services/passport.js

```diff
@@ -151,27 +148,39 @@ passport.use(new GoogleStrategy({
     passReqToCallback: true
   },
   function(request, accessToken, refreshToken, profile, done) {
-    console.log("google Come");
-    console.log(profile);
-
     process.nextTick(function() {
-
       findByUsername(profile.id, function(err, userdb) {
 
         if (err)
           return done(null, err);
         if (!userdb) {
+	  var role = 1;
```

Comment:
This code is not formatted, we are using editorconfig. You should find a plugin for your editor and format the code.